### ISSUE TITLE
SyntaxNet server that does not recreate models (stays alive) 

### ISSUE THREAD

**Issue by malakhovks** [ljm625/syntaxnet-rest-api#5]
It will be great to speed up time of analysis with [TensorFlow Serving](https://github.com/tensorflow/serving)  use.
I find some info about how to implement it 

- [Tensorflow serving and syntaxnet](https://github.com/dsindex/syntaxnet/blob/master/README_api.md) 
- [Standalone SyntaxNet server that does not recreate models (stays alive)](https://github.com/IINemo/docker-syntaxnet_rus)

It's not an issue, but it will great enhancement and feature :)

**Comment by ljm625** [comment]
thanks for the info! will check it out next week :D

**Comment by ljm625** [comment]
Hi there, I just did some research these days, it seems that syntaxnet is introducing DRAGNN which I can use to build native version of the wrapper and also it can process instantly.
The Tensorflow serving is not officially supported and you need to retrain these modules, so I will probably go for the DRAGNN, hopefully I can complete it in next week :D ( pretty busy these days )

**Comment by malakhovks** [comment]
Yes, I'm keeping a close eye on the [Google Research Blog](https://research.googleblog.com/) 
Maybe it makes sense to divide repository into two sub-packages as it is done by Google:
1. **DRAGNN**
2. **SyntaxNet**

To make **2 Docker images for each technology**.

**Comment by ljm625** [comment]
Please check the DRAGNN branch, it's currently using DRAGNN package and pretty fast :D

**Comment by malakhovks** [comment]
Ok, I will test it and let you know

**Comment by ljm625** [comment]
thanks, really appreciate it

**Comment by malakhovks** [comment]
#### Hi! 
#### Started testing the new DRAGNN
**First of all:**
1. Сorrect description of the POST request body (сomma is missing)
{
   "strings": ["Google is awesome!","Syntaxnet is Cool"]**,**   
   "tree": true/false
}
2. Response with the result is missing info, such as POS tag
3. Need to change the way of switching to another ParseySaurus model. 
GET request that respond "null" it's not good :)

**I will continue testing**

**Comment by ljm625** [comment]
Thanks for the feedback!
All these issues above should be fixed
